www/helpers.js

```diff
@@ -296,17 +296,15 @@ module.exports = function init(global, jsUtil, cookieHandler, messages, base64,
 
       try {
         // json
-        if (responseType === validResponseTypes[1]) {
-          response.data = response.data === ''
-            ? undefined
-            : JSON.parse(response.data);
+        if (responseType === validResponseTypes[1] && typeof response.data == 'string') {
```

Comment:
This syntax error is thrown if you attempt to parse a JSON string that has already been parsed into a JavaScript object. i.e. The JSON.parse() method cannot be used to parse something that has already been parsed. It expects a JSON string and if you give it anything else, your code will fail. All in all, it depends on what content type the server is returning. In some cases, the server will return Content-Type: application/json. In other cases, it will return a valid JSON string with some other Content-Type

[source](https://thisinterestsme.com/fix-unexpected-token-o-in-json/#:~:text=This%20syntax%20error%20is%20thrown,that%20has%20already%20been%20parsed.)